apps/prairielearn/src/migrations/20230607155228_pl_courses__help_link__add.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE pl_courses ADD COLUMN IF NOT EXISTS help_link_enabled boolean NOT NULL DEFAULT False;
```

Comment:
Feature flags have moved from columns to a separate process, check 
https://github.com/PrairieLearn/PrairieLearn/blob/bc63ab17a0490f2b2221d5fe25a7da76c6485809/apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/instanceQuestion.js#L37-L40
https://github.com/PrairieLearn/PrairieLearn/blob/bc63ab17a0490f2b2221d5fe25a7da76c6485809/apps/prairielearn/src/lib/features/index.ts#L3

Reply:
Out of curiosity, does something this minor really need to be behind a feature flag? Not sure what the convention is for this type of thing.

Reply:
Good point, I think it makes sense to enable it by default. We've done more breaking changes in the past without a feature flag.

Reply:
Yeah, users won't even see a change unless they have the help links added anyway, and nobody has them yet because the feature is new 😅